released/learning_journey/btp200/main.tf

```diff
@@ -0,0 +1,164 @@
+
+
+###############################################################################################
+# This is the Terraform script for the BTP_200 Learning Journey. In this script you will create
+# the infrastructure for the development of an SAP extension project  
+# The script will do the following
+#   - create a new subaccount (if the subaccount id is not set)
+#   - add users as subaccount administrators and viewers
+#   - create entitlements for the following services:
+#        * SAP Business Application Studio
+#        * SAP Continous & Integration Application
+#        * SAP Build Workzone - standard edition
+#   - create subscriptions
+#   - add user to service role collections
+###############################################################################################
+
+###############################################################################################
+# Setup subaccount domain and the CF org (to ensure uniqueness in BTP global account)
+###############################################################################################
+locals {
```

Comment:
Would it make sense to add a UUID as in the discovery center missions to ensure the uniqueness of the domain